### ISSUE TITLE
[app] Network request failed on iOS

### ISSUE THREAD

**Issue by matart15** [codefoundries/UniversalRelayBoilerplate#285]
I can't run it on ios simulator.
I run `yarn dev` can see `http://10.0.1.2:8080/` display folder contents, `http://10.0.1.2:4444/` display web, `http://10.0.1.2:8081/` display `React Native packager is running.`. from ios simulator safari.

But still get `Network request failed`
chrome console display this. But no XHR network request sent.
```
Running application UniversalRelayBoilerplate ({
    initialProps =     {
    };
    rootTag = 1;
})
Running application "UniversalRelayBoilerplate" with appParams: {"rootTag":1,"initialProps":{}}. __DEV__ === true, development-level warning are ON, performance optimizations are OFF
XXX layout={"width":375,"height":667,"contentTopMargin":64}
XXX navigationBarStyle={"paddingTop":0,"height":64,"borderBottomWidth":0,"backgroundColor":"#03A9F4"}
XXX navigationBarTitleStyle={"top":0,"color":"#000000"}
Network request failed
```

what i did is

```
git clone https://github.com/codefoundries/UniversalRelayBoilerplate
cd UniversalRelayBoilerplate
yarn
yarn setup-local
yarn update-ip
yarn dev

yarn dev v0.18.1
$ rm -rf build && concurrently --kill-others "npm run dev-webpack" "npm run dev-server" "npm run start"
[1]
[1] > UniversalRelayBoilerplate@14.4.2 dev-server /Users/matar/htdocs/test/UniversalRelayBoilerplate
[1] > better-npm-run dev-server
[1]
[2]
[2] > UniversalRelayBoilerplate@14.4.2 start /Users/matar/htdocs/test/UniversalRelayBoilerplate
[2] > node node_modules/react-native/local-cli/cli.js start --reset-cache
[2]
[0]
[0] > UniversalRelayBoilerplate@14.4.2 dev-webpack /Users/matar/htdocs/test/UniversalRelayBoilerplate
[0] > rm -rf build && better-npm-run dev-webpack
[0]
[0] running better-npm-run in /Users/matar/htdocs/test/UniversalRelayBoilerplate
[1] running better-npm-run in /Users/matar/htdocs/test/UniversalRelayBoilerplate
[0] Executing script: dev-webpack
[0]
[1] Executing script: dev-server
[1]
[0] to be executed: ./node_modules/.bin/webpack-dev-server --hot --inline --no-info --colors
[1] to be executed: ./node_modules/.bin/nodemon --exec ./node_modules/.bin/babel-node -- ./server/server.js
[1] [nodemon] 1.11.0
[1] [nodemon] to restart at any time, enter `rs`
[1] [nodemon] watching: *.*
[1] [nodemon] starting `./node_modules/.bin/babel-node ./server/server.js`
[0] Running Webpack, process.env.NODE_ENV=undefined, version=14.4.2.123
[2] Scanning 901 folders for symlinks in /Users/matar/htdocs/test/UniversalRelayBoilerplate/node_modules (8ms)
[2]  ┌────────────────────────────────────────────────────────────────────────────┐
[2]  │  Running packager on port 8081.                                            │
[2]  │                                                                            │
[2]  │  Keep this packager running while developing on any JS projects. Feel      │
[2]  │  free to close this tab and run your own packager instance if you          │
[2]  │  prefer.                                                                   │
[2]  │                                                                            │
[2]  │  https://github.com/facebook/react-native                                  │
[2]  │                                                                            │
[2]  └────────────────────────────────────────────────────────────────────────────┘
[2] Looking for JS files in
[2]    /Users/matar/htdocs/test/UniversalRelayBoilerplate
[2]
[0]  http://10.0.1.2:8080/
[0] webpack result is served from http://10.0.1.2:8080/14.4.2.123/
[0] content is served from /Users/matar/htdocs/test/UniversalRelayBoilerplate
[2] [1/19/2017, 5:25:13 PM]         HMR Server listening on /hot
[2]
[2] React packager ready.
[2]
[1] info: Starting application name=urb, version=14.4.2.123, NODE_ENV=development, HOST=10.0.1.2, PORT=4444, PUBLIC_URL=http://10.0.1.2:4444, process_title=/Users/matar/.nvm/versions/node/v6.9.1/bin/node, process_pid=13560, local_ip=10.0.1.2
[1] 🚀 Initializing persisters - start ...
[1] 🏆 Initializing Cassandra persister - success.
[1] ☄ DEVELOPMENT. Server listening on 127.0.0.1
[1] ☄ DEVELOPMENT. Server listening on 10.0.1.2
[2] [1/19/2017, 5:25:15 PM] <START> Initializing Packager
[2] [1/19/2017, 5:25:15 PM] <START> Building in-memory fs for JavaScript
[2] [1/19/2017, 5:25:15 PM] <END>   Building in-memory fs for JavaScript (307ms)
[2] [1/19/2017, 5:25:15 PM] <START> Building Haste Map
[2] [1/19/2017, 5:25:16 PM] <END>   Building Haste Map (493ms)
[2] [1/19/2017, 5:25:16 PM] <END>   Initializing Packager (907ms)
[2] [1/19/2017, 5:26:38 PM] <END>   Transforming files (13166ms)
[2] [1/19/2017, 5:26:38 PM] <END>   Requesting bundle bundle_url: /index.ios.bundle?platform=ios&dev=true&minify=false (13601ms)
[2] [1/19/2017, 5:26:38 PM] <START> Requesting bundle bundle_url: /index.ios.bundle?platform=ios&dev=true&minify=false
[2] [1/19/2017, 5:26:38 PM] <END>   Requesting bundle bundle_url: /index.ios.bundle?platform=ios&dev=true&minify=false (18ms)

```

**Comment by thelordoftheboards** [comment]
Can you check if the app network security is configured?

https://ste.vn/2015/06/10/configuring-app-transport-security-ios-9-osx-10-11/

**Comment by thelordoftheboards** [comment]
Also, are you running from XCode?

**Comment by matart15** [comment]
I am running `react-native run-ios`. I will try XCode with NSAppTransportSecurity

**Comment by matart15** [comment]
@thelordoftheboards thank you man. It worked.

**Comment by thelordoftheboards** [comment]
@matart15 Glad it worked for you, I am reopening the issue to make sure it works for everyone else who downloads the repo.

**Comment by thelordoftheboards** [comment]
Should be fixed in 15.0. If anyone else testes it and there is still an issue please reopen.
